src/ui/i18n-anchor.tsx

```diff
@@ -0,0 +1,23 @@
+import {
+	A as RouterAnchor,
+	useLocation,
+	type AnchorProps,
+} from "@solidjs/router";
+import { Show } from "solid-js";
+import { getLocaleFromPathname, isValidLocale } from "~/i18n";
+
+export function A(props: AnchorProps) {
+	const { pathname } = useLocation();
+	const locale = () => getLocaleFromPathname(pathname);
+
+	return (
+		<Show when={isValidLocale(locale())} fallback={<RouterAnchor {...props} />}>
```

Comment:
just to make sure I got the behaviour right, at the moment this means that if the language is in the `SUPPORTED_LOCALES` array any click of the nav after a non implemented section will continue going to another non implement section right?

Here is the behaviour that is happening right now:
- go to `/pt-br/quick-start`
- displays a "Esta seção ainda não foi traduzida." message
- click Intro to Reactivity
- routes to `/pt-br/concepts/intro-to-reactivity``
- displays again a "Esta seção ainda não foi traduzida." message

Reply:
yes. To prevent that (and only linking to existing content) we'd have to create a different navigation to each supported locale and dynamically import the modules for the SideBar when locale is changed.

It's not impossible, perhaps this can be implemented in a follow-up PR. But I think we need to consider first the impact that would have on UX (when some links are in the selected language and others are in English)